### DISCUSSION TITLE
Python

### DISCUSSION THREAD

**Discussion by Adamgil16** [doocs/leetcode#3187] [General]
Input: 1,2,3,4,5 
Result : 15

**Comment by Kouroshdan** [comment]
Hey Just to make sure I got what you meant you want all the enputs to be added ? 
like 1 + 2 + 3 + 4 +5 and then It  will print 15 ?
If you want I can help you with that

**Comment by VyvaHart** [comment]
Hi @Adamgil16 

```
def sum_numbers(numbers):
      return sum(numbers)

numbers = [1, 2, 3, 4, 5]
result = sum_numbers(numbers)
print(result)  # Output: 15
```

**Comment by EmrD** [comment]
You can use sum function. For example; 
`numbers=[1,2,3,4,5]`
`print(sum(numbers))`
